sonic_platform_base/module_base.py

```diff
@@ -68,6 +86,83 @@ def get_system_eeprom_info(self):
         """
         raise NotImplementedError
 
+    def get_name(self):
```

Comment:
This class inherits from DeviceBase, which already defines a `get_name()` method.

Reply:
Added this mainly to highlight the docstring since we need names in certain standard format. If not, we can remove it. Please suggest.